### ISSUE TITLE
🚀 Feature Request: Responsive View Default Dimensions

### ISSUE THREAD

**Issue by hiviz** [elementor/elementor#19016]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Inside the Elemenetor Editor, each responsive view is displayed with modifiable dimensions: https://prnt.sc/Xf1rZV_ZzmIX. These dimensions are set to some default value. These default dimensions are not always suitable.

### Describe the solution you'd like

Option for one to change the default values for each responsive view display dimensions.

### Describe alternatives you've considered

If one wants to see the particular responsive view in different dimensions, one must manually set it by overriding the current (default) value. This is incredibly laborious.

**Comment by rami-elementor** [comment]
Hi @hiviz ,

Are you talking about the "**Site Settings**" > "**Layout**" > "**Breakpoints**" ?

![Elementor-Elementor-6 (1)](https://user-images.githubusercontent.com/92088692/177029793-3fba8ce6-9e02-41e9-a41d-87f06f8bd466.png)

**Comment by hiviz** [comment]
No, I'm talking about the modifiable display dimensions in the top right (once "responsive view" option is selected), as can be seen in the screenshot I attached in the opening ticket, repeated here for your benefit: https://prnt.sc/Xf1rZV_ZzmIX

**Comment by rami-elementor** [comment]
Source: https://github.com/elementor/elementor/blob/master/assets/dev/js/editor/editor-base.js#L654-L697

**Comment by hiviz** [comment]
Can you please provide context to your link?

**Comment by rami-elementor** [comment]
@hiviz , if we prioritize this task, the developers will have a good idea where to start.

**Comment by rami-elementor** [comment]
@hiviz it's a feature request. It doesn't exist yet. In the future, if we decide to develop this feature, the developer will see the link and know where to start.

**Comment by hiviz** [comment]
@rami-elementor so now we just wait and hope for the developers to pick up this ticket and implement the feature request?

**Comment by rami-elementor** [comment]
@hiviz basically our product team decides which features are prioritized and developed. We have our own development roadmap and our own goals and product vision.

Please note that there are hundreds of feature request, we can't develop everything. We are not a development agency, we are a product company. Some of the requests will be rejected.

We add features after an extensive research and features with height impact. If we see a high demand for a GitHub feature, we will update our roadmap.

**Comment by PowerUser1234** [comment]
If the option was even there in some form of a text file, per page or per total project, to edit Breakpoint pixel values by one Height value, or by Height and Width pixel values, and to introduce your own Breakpoints, which is pretty necessary for the three or four screen dimensions which are needed for multiplatform, it might be enough.  At any rate, for me, I believe that such an active feature should be made available and incorporated into the next WordPress broadcast update release for free Elementor, past version 3.6.7, which is where we are at this time.

People will need the following.  Most of the time you can get away with painting to Widescreen centrally, as thought it were "square" screen, and so at some bare minimum, three options may be enough, but in the end you really need four breakpoints included that can be edited for proper values:

Mobile, Width and Height, or Height.
Tablet, Width and Height, or Height.
"Square" Desktop, Width and Height, or Height.
WideScreen Desktop, Width and Height, or Height.

I can't edit these values in a .ini file, because they are compiled and sealed away inside the free Elementor plugin compile, within editor-base.js.  I am not in a position, able to, edit and rebuild free Elementor itself you see.  At any rate, a compatability and ubiquity change would be the thing to do, and it strikes me that this needn't be terribly hard for the programmers/coders involved.  Could such changes please be implemented?

**Comment by rami-elementor** [comment]
@PowerUser1234 We understand the benefits of this feature request. But our product teams also need to compare that to other feature requests. We are looking for features with high demand and with the potential to impact high number of users.
